_beats/libbeat/_meta/fields.common.yml

```diff
@@ -25,3 +24,13 @@
           type: keyword
           description: >
             Error type.
+
+    - name: beat.name
+      type: alias
+      path: host.name
+      migration: true
+
+    - name: beat.hostname
```

Comment:
This is a good reminder to review all of these aliases that we inherit and decide if  they should be kept (they're not new)